Templates/GenericWebHook-CSharp/function.json

```diff
@@ -5,6 +5,7 @@
             "type": "httpTrigger",
             "direction": "in",
             "webHookType": "genericJson",
+            "authLevel": "function",
```

Comment:
Right now webhook doesn't have a separate trigger type. So portal will have to have a mechanism to only show auth level setting when webhook type is selected.